### ISSUE TITLE
possible bug in batchnorm1d

### ISSUE THREAD

**Issue by tujiaojiao1981** [BGIResearch/BioLLM#1]
self.GCN_layers1 = GraphConv(drug_input_dim, drug_hidden_dim)   # first GCN
        self.drug_batchnorm1 = nn.BatchNorm1d(num_features=100)
        self.GCN_layers2 = GraphConv(drug_hidden_dim, drug_hidden_dim)  # middle GCN
        self.drug_batchnorm2 = nn.BatchNorm1d(num_features=100)
        self.final_GCN_layers = GraphConv(drug_hidden_dim, drug_concate_before_dim)
        self.drug_batchnorm3 = nn.BatchNorm1d(num_features=100)
        self.global_max_pooling1d = nn.AdaptiveMaxPool1d(1)

I read through the above codes for drug encoding. I wonder why the num_features are always 100 in batchNorm1d? It should be drug_hidden_dim for the first two if I understand correct.
